### ISSUE TITLE
Should it be possible to use the simple_trainer CLI path on data process by ns-process-data?

### ISSUE THREAD

**Issue by vincentwoo** [nerfstudio-project/gsplat#505]
I'm trying to run simple_trainer directly on a dataset produced by realitycapture and converted by `ns-process-data`, however, the output of which is simply a `transforms.json` and a `.ply`. It seems that simple_trainer only accepts the COLMAP sparse format, which is somewhat a pain to produce.

**Comment by maturk** [comment]
Yeah a dataparser supporting the .json format would have to be implemented.
